packages/client-readmodel-writer/kubernetes/dev/configmap.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: interop-be-client-readmodel-writer
+  namespace: dev-refactor
+data:
+  KAFKA_CLIENT_ID: "dev-pagopa-interop-client"
+  KAFKA_GROUP_ID: "client-readmodel-writer"
+  KAFKA_BROKERS: "boot-yqksbq44.c3.kafka-serverless.eu-central-1.amazonaws.com:9098"
+  AUTHORIZATION_TOPIC: "event-store.authorization.events"
```

Comment:
```suggestion
  AUTHORIZATION_TOPIC: "event-store.authz.events"
```

Reply:
Updated to `"event-store.dev-refactor_authz.events"`